src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `styleObject`. Although the name is not as generic as `object`, it could be more descriptive of its purpose, such as `stylesMap` or `cssProperties`.